api/coreservice.go

```diff
@@ -1213,6 +1217,58 @@ func (core *coreService) getBlockByHeight(height uint64) (*apitypes.BlockWithRec
 	}, nil
 }
 
+func (core *coreService) BlobSidecarsByHeight(height uint64) ([]*apitypes.BlobSidecarResult, error) {
+	header, err := core.bc.BlockHeaderByHeight(height)
+	if err != nil {
+		return nil, err
+	}
+	res := make([]*apitypes.BlobSidecarResult, 0)
+	blobs, txHashes, err := core.getBlobSidecars(height)
+	if err != nil {
+		return nil, err
+	}
+	txIndices := make([]uint64, 0)
+	for _, txHash := range txHashes {
+		_, _, index, err := core.ActionByActionHash(txHash)
+		if err != nil {
+			return nil, err
+		}
+		txIndices = append(txIndices, uint64(index))
+	}
+	blkHash := header.HashBlock()
+	for i, blob := range blobs {
```

Comment:
move L1232 to here, get the `index` and use it at L1244, then we don't need to define `txIndices`?

Reply:
fixed